AngularWebUI/ClientApp/src/app/relations/add-edit-relation/add-edit-relation.component copy.html

```diff
@@ -0,0 +1,67 @@
+<div class="form-froup row">
+  <label class="col-sm-2 col-form-label" *ngIf="relation.Id!= null">Id</label>
+  <div class="col-sm-2" *ngIf="relation.Id!= null">
+    <input type="text" readonly class="form-control" [(ngModel)]="Id" />
+  </div>
+
+  <label class="col-sm-2 col-form-label">Name</label>
+  <div class="col-sm-2">
+    <input type="text" aria-required="true" class="form-control" required minlength="3" [(ngModel)]="Name" #name="ngModel" />
+  </div>
+
+  <div *ngIf="name.invalid"
+       class="alert alert-danger">
+    <div *ngIf="name.errors.required">
+      Name is required.
+    </div>
+    <div *ngIf="name.errors.minlength">
+      Name must be at least 3 characters long.
```

Comment:
Why name with less then 3 chars is not valid? Use valiadation on maxlength f.e. 50 charactes.